src/main/java/uk/gov/hmcts/reform/pip/data/management/service/helpers/LanguageResourceHelper.java

```diff
@@ -41,7 +42,9 @@ public static Map<String, Object> getLanguageResources(ListType listType, Langua
 
     private static Map<String, Object> readResources(ListType listType, Language language) throws IOException {
         String resourceName = UPPER_UNDERSCORE.to(LOWER_CAMEL, listType.name());
-        return readResourcesFromPath(resourceName, language);
+        Map<String, Object> languageResources = new ConcurrentHashMap<>();
+        languageResources.putAll(readResourcesFromPath(resourceName, language));
```

Comment:
Can I check if this copy is definitely needed? If not, readResourcesFromPath looks to return the same type of Map so I believe that can be used directly

Reply:
I see, makes sense.